src/ios/CDVInAppBrowser.m

```diff
@@ -1163,8 +1271,6 @@ + (CDVInAppBrowserOptions*)parseOptions:(NSString*)options
 			NSString* value_lc = [value lowercaseString];
 			
 			BOOL isBoolean = [value_lc isEqualToString:@"yes"] || [value_lc isEqualToString:@"no"];
-			NSNumberFormatter* numberFormatter = [[NSNumberFormatter alloc] init];
-			[numberFormatter setAllowsFloats:YES];
```

Comment:
After moving all default value settings to init, you will have to keep these lines